PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## RALLY STORY
```

Comment:
Should not include the rally story here. We may get pull requests from non employees as this is currently public and open sourced.